### PR TITLE
Valkyrie models

### PR DESCRIPTION
**Pull request by kirkkwang**
# Story

Ref:
- https://github.com/scientist-softserv/ams/issues/36

Generate valkyrie models of the following:
- Asset
- PhysicalInstantiation
- DigitalInstantiation
- EssenceTrack
- Contribution

Now we have:
- AssetResource
- PhysicalInstantiationResource
- DigitalInstantiationResource
- EssenceTrackResource
- ContributionResource

# Notes

Even after registering the test_adapter I was getting still getting errors so I'm skipping all the generated model test for now.
https://github.com/samvera/hyrax/blob/4.x-stable/spec/test_app_templates/lib/generators/test_app_generator.rb#L219-L228

### CODE REVIEW

**Comment by ShanaLMoore** [comment]
Should we upgrade Hyrax too or has that already been done?

**Comment by kirkkwang** [comment]
That's gonna be tricky with our dual boot situation, Hyrax `main` depends on Ruby 3.2 or higher.

**Comment by ShanaLMoore** [comment]
> That's gonna be tricky with our dual boot situation, Hyrax `main` depends on Ruby 3.2 or higher.

Oh right, we put a pause on that for a moment. Hmm 🤔 This looks fine for me but wonder if we should get Rob's eyes on it.

File: config/metadata/asset_resource.yaml [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+    type: string
+    multiple: true
+    index_keys:
+      - "series_description_tesim"
+    form:
+      required: false
+      primary: false
+      multiple: true
+  intended_children_count:
+    type: string
```
Comment (laritakr) [inline config/metadata/asset_resource.yaml:343]: This needs to include index_keys

Comment (kirkkwang) [reply config/metadata/asset_resource.yaml:343]: additional indexing will be handled in another issue

**Review by orangewolf — APPROVED** [review]

**Pull request merged**